webapp/sources/rudder/rudder-web/src/main/style/rudder/rudder-bootstrap.scss

```diff
@@ -97,4 +99,17 @@ $link-color : $rudder-txt-link;
   label{
     font-weight: bold;
   }
+}
+// -- CHECKBOX/RADIO OVERRIDE
+.checkbox, .radio {
+  & {
+    position: relative;
+    display: block;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+
+  & label {
+    cursor: pointer;
+  }
```

Comment:
those are declared in bootstrap 3, disappeared completely in bootstrap 5 :neutral_face: